RendererPlugins/PBRT/RunPBRT.m

```diff
@@ -65,17 +65,57 @@
 output = fullfile(renderings, [sceneBase '.dat']);
 
 %% Invoke PBRT.
-% set the dynamic library search path
-[newLibPath, originalLibPath, libPathName] = SetRenderToolboxLibraryPath();
 
-% find the PBRT executable
```

Comment:
Nice.  This is cool too.

May I propose 3 changes?  I think these are just fussy organizational things.
 1. How about moving the `dockerFlag` into the `pbrt` struct, rather than `hints` struct.  This would keep the Docker choice local to pbrt.  We could think of the `dockerFlag` as roughly on the same level as the pbrt executable path, which is already in the `pbrt` struct.
 2. Instead of a true/false `dockerFlag`, what about a string field like `dockerImageName`?  Then the literal value of `vistalab/pbrt` could be broken out and configured using this field instead of hard-coded into the function.  Users could leave the field empty if they want to use local pbrt.
 3. What about moving the `docker pull` command into the RTB configuration script, rather than `RunPBRT()` itself?

If you're on board with these changes, you could edit [RenderToolbox3ConfigurationTemplate](https://github.com/DavidBrainard/RenderToolbox3/blob/master/Utilities/RenderToolbox3ConfigurationTemplate.m) to add expected fields to the `pbrt` struct and put the `docker pull` command there.